### ISSUE TITLE
DocumentType probably has wrong signature

### ISSUE THREAD

**Issue by maksimkurb** [vknet/vk#781]
**Describe the bug**
`VkNet.Model.Attachments.Document` has field `DocumentType Type`
According to [documentation](https://vk.com/dev/objects/doc) Type should be integer (enum)
Also `DocumentType` has strange fields: `Id` and `Name` for gifts and `Count` for documents. It's really confusing.

Because of this, I can not for ex. distinguish text document from animation

**Expected behavior**
`DocumentType` should be enum with this values:
```
1 - text documents;
2 - archives;
3 - gif;
4 - images;
5 - audio;
6 - video;
7 - e-books;
8 - unknown.
```

**Additional context**
Using VkBot and group polling token to receive messages and parse attachments.

**Comment by itsokto** [comment]
@maksimkurb 
https://github.com/vknet/vk/blob/59b7de46883a5a1af40ef8f82aadd31f9b4fb3f4/VkNet/Model/Attachments/Document.cs#L50

**Comment by maksimkurb** [comment]
Oh, so it fixed in 1.41.0
Excuse for troubling
